core/src/main/java/com/linecorp/armeria/client/ClientFactoryOptions.java

```diff
@@ -263,6 +263,26 @@ private static long clampedDefaultMaxClientConnectionAge() {
                 return newValue.option().newValue(builder.build());
             });
 
+    /**
+     *  The number of bytes per sec allowed to be read for each channel.
+     *  A value of {@code 0} signifies no limit is applied.
+     */
+    public static final ClientFactoryOption<Long> WRITE_BYTES_PER_SEC_LIMIT =
```

Comment:
Naming: We usually start an option name with `maxXXX` and `minXX` when the option limits something.
Should we change the naming style to `MAX_WRITE_BYTES_PER_SEC`?